__tests__/integration.test.js

```diff
@@ -85,6 +85,56 @@ describe("/api/articles/:article_id", () => {
         expect(message).toBe("Bad request");
       });
   });
+  test("PATCH 200 - Responds with the updated article and the votes count changed by given amount", () => {
+    const send = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(send)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
```

Comment:
if we ever update the test data, this test will unfortunately break - try checking the types of the rest of the properties rather than their specific values if you'd really like to test them. Another matcher that may be useful to look into: .toMatchObject() rather than toEqual which is a much stricter matcher